File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
a = list(map(int, input().split()))
data = [1, 1, 2, 2, 2, 8]
result = []

for i in range(len(a)):
  if a[i] != data[i]:
    result.append((data[i]-a[i]))
  else:
    result.append(0)
print(*result)

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.rb
# set = [1,1,2,2,2,8]
# input = gets.split.map(&:to_i)
# result = []

# (0..5).each do |i|
#   result << (set[i] - input[i])
# end

# puts result.join(" ")

set = [1,1,2,2,2,8]
input = gets.split.map(&:to_i)

result = set.map.with_index { |value, i| value - input[i] }
puts result.join(" ")
